deploy/image/csi-image-plugin.yaml

```diff
@@ -0,0 +1,90 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-imageplugin
+spec:
+  selector:
+    matchLabels:
+      app: csi-imageplugin
+  template:
+    metadata:
+      labels:
+        app: csi-imageplugin
+    spec:
+      hostNetwork: true
+      containers:
+        - name: node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
+          imagePullPolicy: Always
```

Comment:
is this necessary?

Reply:
not sure I follow? The imagePullPolicy or the sidecar? I think the sidecar's needed isnt it?

Reply:
i mean the imagePullPolicy: always

Reply:
Ah. Yeah, I can update that to IfNotPresent